app/src/main/java/com/example/concordia_campus_guide/Helper/DrawDirectionsPolyLines.java

```diff
@@ -0,0 +1,44 @@
+package com.example.concordia_campus_guide.helper;
```

Comment:
File name shouldn't start with a verb. DirectionsPolylinesDrawer is better in my opinion

Reply:
done